pubsub/api/pubsub.php

```diff
@@ -138,6 +138,108 @@
         }
     });
 
+    $application->add(new Command('dead-letter'))
+        ->setDescription('Manage Dead Letter Policies for Pub\Sub')
+        ->setHelp(<<<EOF
+    The <info>%command.name%</info> command manages Pub\Sub dead letter policies.
+
+    <info>php %command.full_name% create --project my-project
+        --topic my-topic
+        --subscription my-subscription
+        --dead-letter-topic my-dead-letter-topic</info>
+
+    <info>php %command.full_name% update --project my-project
+        --topic my-topic
+        --subscription my-subscription
+        --dead-letter-topic my-dead-letter-topic</info>
+
+    <info>php %command.full_name% remove --project my-project
+        --topic my-topic</info>
+
+    <info>php %command.full_name% pull --project my-project
+        --topic my-topic --subscription my-subscription</info>
+
+    EOF
```

Comment:
Getting a kokoro syntax error for PHP 7.2 on this line. This needs to be flush with no indentation
```suggestion
EOF
```